redis/README.md

```diff
@@ -0,0 +1,28 @@
+# Redis packages in Baseplate.go
+
+We support two different sets of Redis packages in Baseplate.go.
+
+## Use Redis as a cache
+
+If you use Redis as a cache, it's recommended to use
+[`redispipebp`][redispipebp] + [`redisx`][redisx] packages.
+They are built on top of [`redispipe`][redispipe] open source project.
+
+This combination can also be used if you use Redis as a DB but only do readings
+and not writings.
+
```

Comment:
you mention in the DB section why that's good for the DB use-case, but you don't mention why redispipe is good for the cache use-case.

Just a single sentence is probably fine, like "redispipe has improved support for pipelining reads to maximize throughput" or something (I'm making that up, I don't know if that's what it is)

Reply:
I'll wait for @pacejackson to provide a description :)